src/var_equality.jl

```diff
@@ -57,16 +57,19 @@ function Base.show(io::IOContext, t::VarianceEqualityTest)
     end
 end
 
-function anova(scores::AbstractVector{<:Real}...)
-    Nᵢ = [length(g) for g in scores]
+function anova(scores::AbstractVector{<:AbstractVector})
+    Nᵢ = length.(scores)
     Z̄ᵢ = mean.(scores)
-    Z̄ = mean(Z̄ᵢ)
-    SStᵢ = Nᵢ .* (Z̄ᵢ .- Z̄).^2
-    SSeᵢ = sum.( (z .- z̄).^2 for (z, z̄) in zip(scores, Z̄ᵢ) )
+    Z̄ = dot(Z̄ᵢ, Nᵢ) / sum(Nᵢ)
+    SStᵢ = Nᵢ .* abs2.(Z̄ᵢ .- Z̄)
+    SSeᵢ = [sum(z -> abs2(z - z̄ᵢ), scoresᵢ) for (scoresᵢ, z̄ᵢ) in zip(scores, Z̄ᵢ)]
```

Comment:
Could use a `map` here as well? This would also be better for tuple or static array input.
```suggestion
    SSeᵢ = map((scoresᵢ, z̄ᵢ) -> sum(z -> abs2(z - z̄ᵢ), scoresᵢ), scores, Z̄ᵢ)
```

Reply:
Same thing as for `Nᵢ` -- `SSeᵢ` has to be a vector